content/en/docs/ops/ambient/getting-started/test.sh

```diff
@@ -65,9 +65,12 @@ _verify_contains snip_verify_traffic_notsleep_to_productpage "$snip_verify_traff
 snip_adding_your_application_to_the_ambient_mesh_1
 
 # test traffic after ambient mode is enabled
-_verify_contains snip_adding_your_application_to_the_ambient_mesh_2 "$snip_adding_your_application_to_the_ambient_mesh_2_out"
-_verify_contains snip_adding_your_application_to_the_ambient_mesh_3 "$snip_adding_your_application_to_the_ambient_mesh_3_out"
+_verify_like snip_adding_your_application_to_the_ambient_mesh_1 "$snip_adding_your_application_to_the_ambient_mesh_1_out"
+_verify_contains snip_adding_your_application_to_the_ambient_mesh_2 ""
```

Comment:
Just trying to do a Quick Look and do to renumbering, some of these may be different:

This line is just doing a label, so no verify needed:
```suggestion
snip_adding_your_application_to_the_ambient_mesh_2
```